src/deployments.ts

```diff
@@ -42,62 +44,71 @@ export const DEPLOYMENT_SYLVESTER_POLYGON_MAINNET_V0 = {
   network: NETWORK_POLYGON_MAINNET,
   contractType: RenftContractType.SYLVESTER,
   version: SylvesterVersion.V0,
+  startBlock: 28399140,
 } as const;
 
 export const DEPLOYMENT_SYLVESTER_POLYGON_MAINNET_V1 = {
   contractAddress: '0x4e52b73aa28b7ff84d88ea3a90c0668f46043450',
   network: NETWORK_POLYGON_MAINNET,
   contractType: RenftContractType.SYLVESTER,
   version: SylvesterVersion.V1,
+  startBlock: 36825974,
 } as const;
 
 export const DEPLOYMENT_WHOOPI_AVALANCHE_FUJI_TESTNET_V0 = {
   contractAddress: '0x42816FA3cB0aDc3fcAdED3109323c0Bc19215084',
   network: NETWORK_AVALANCHE_FUJI_TESTNET,
   contractType: RenftContractType.WHOOPI,
   version: WhoopiVersion.V0,
+  startBlock: 11500156,
 } as const;
 
 export const DEPLOYMENT_WHOOPI_AVALANCHE_MAINNET_V0 = {
   contractAddress: '0x6Ee495ecEd3A0255057667FF2685e53f54A19A65',
   network: NETWORK_AVALANCHE_MAINNET,
   contractType: RenftContractType.WHOOPI,
   version: WhoopiVersion.V0,
+  startBlock: 19408332,
```

Comment:
Can you share where these values come from? I can't find these values:
https://snowtrace.io/txs?a=0x6Ee495ecEd3A0255057667FF2685e53f54A19A65&p=2000

Reply:
The contract was created on this transaction: https://snowtrace.io/tx/0x5c913c9d83b72784199b26e2c46c1cad184960df1c1f74754d747fc32b9aa129